src/auth/services/team.service.ts

```
@@ -130,75 +148,95 @@ export class TeamService {
-    return this.getTeamById(teamId, userId)
+    return this.getTeamById(teamId, userId);
   }
 
-  async inviteTeamMember(teamId: string, inviterId: string, inviteDto: InviteTeamMemberDto): Promise<TeamMember> {
-    const team = await this.getTeamById(teamId, inviterId)
+  async inviteTeamMember(
+    teamId: string,
+    inviterId: string,
+    inviteDto: InviteTeamMemberDto,
+  ): Promise<TeamMember> {
+    const team = await this.getTeamById(teamId, inviterId);
```

ESLint

[error] 'team' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)